frontend/packages/schema-model/src/lib/mutations/ui-schema-reducers.ts

```diff
@@ -6,266 +6,284 @@ import {
 };
-export const addRootItem: UiSchemaReducer<AddRootItemArgs> =
-  (uiSchema: UiSchemaNodes, { location, name, props, callback }) => {
-    const newSchema = deepCopy(uiSchema);
-    const newPointer = getUniqueNodePath(newSchema, [location, name].join('/'));
-    const newNode = createNodeBase(newPointer);
-    newNode.implicitType = false;
-    newSchema.push(Object.assign(newNode, props));
-    getNodeByPointer(newSchema, ROOT_POINTER).children.push(newPointer);
-    callback(newPointer);
-    return newSchema;
-  };
+export const addRootItem: UiSchemaReducer<AddRootItemArgs> = (
+  uiSchema: UiSchemaNodes,
+  { location, name, props, callback },
+) => {
+  const newSchema = deepCopy(uiSchema);
+  const newPointer = getUniqueNodePath(newSchema, [location, name].join('/'));
+  const newNode = createNodeBase(newPointer);
+  newNode.implicitType = false;
+  newSchema.push(Object.assign(newNode, props));
+  getNodeByPointer(newSchema, ROOT_POINTER).children.push(newPointer);
+  callback(newPointer);
+  return newSchema;
+};
 
 export type AddPropertyArgs = {
   pointer: string;
   props: Partial<UiSchemaNode>;
   callback?: (pointer: string) => void;
 };
-export const addProperty: UiSchemaReducer<AddPropertyArgs> =
-  (uiSchema: UiSchemaNodes, { pointer, props, callback }) => {
-    const newSchema = deepCopy(uiSchema);
-    const addToNode = getNodeByPointer(newSchema, pointer);
-    const pointerBase = addToNode.isArray
-      ? makePointer(addToNode.pointer, Keyword.Items)
-      : addToNode.pointer;
-    const newNodePointer = getUniqueNodePath(
-      newSchema,
-      makePointer(pointerBase, Keyword.Properties, 'name')
-    );
-    addToNode.children.push(newNodePointer);
-    callback && callback(newNodePointer);
-    props.implicitType = false;
-    newSchema.push(Object.assign(createNodeBase(newNodePointer), props));
-    return newSchema;
-  };
+export const addProperty: UiSchemaReducer<AddPropertyArgs> = (
+  uiSchema: UiSchemaNodes,
+  { pointer, props, callback },
+) => {
+  const newSchema = deepCopy(uiSchema);
+  const addToNode = getNodeByPointer(newSchema, pointer);
+  const pointerBase = addToNode.isArray
+    ? makePointer(addToNode.pointer, Keyword.Items)
+    : addToNode.pointer;
+  const newNodePointer = getUniqueNodePath(
+    newSchema,
+    makePointer(pointerBase, Keyword.Properties, 'name'),
+  );
+  addToNode.children.push(newNodePointer);
+  callback && callback(newNodePointer);
+  props.implicitType = false;
+  newSchema.push(Object.assign(createNodeBase(newNodePointer), props));
+  return newSchema;
+};
 
 export type DeleteEnumValueArgs = {
   path: string;
-  value: string;
+  index: number;
+};
+export const deleteEnumValue: UiSchemaReducer<DeleteEnumValueArgs> = (
+  uiSchema: UiSchemaNodes,
+  { path, index },
+) => {
+  const newSchema = deepCopy(uiSchema);
+  const enumItem = getNodeByPointer(newSchema, path);
+  enumItem.enum = removeItemByIndex(enumItem.enum, index);
```

Comment:
This line is the only real change in this massive file. 
All other is formatting unfortunately.